sections/annoyances.txt

```diff
@@ -60,6 +60,8 @@ borsonline.hu##.subscription-box
 bpiautosok.hu###onesignal-bell-container
 bpiautosok.hu###onesignal-slidedown-container
 bpiautosok.hu##.extpp-container
+! https://github.com/hufilter/hufilter-dev/issues/367
+budapestkornyeke.hu##.extpp-container
```

Comment:
Maybe `.extpp-container` is covered by `||ex1tp.com^$third-party`

Reply:
thanks, fixed in https://github.com/hufilter/hufilter-dev/pull/368/commits/e2b365c74331f8313ffe677f89546f27613cea2a